examples/react-app/src/index.tsx

```diff
@@ -7,6 +7,8 @@ import { StrictMode } from 'react';
 import router from './router';
 
 import { Density, Mode, applyDensity, applyMode } from '@cloudscape-design/global-styles';
+import '@cloudscape-design/global-styles/index.css';
```

Comment:
Where have you used these styles?

Reply:
it is used in all App kit components in react-components package. But specifically for this app example it is used in Knowledge Graph component.